IS7024/SongkickSchemaAll.json

```diff
@@ -0,0 +1,355 @@
+﻿{
```

Comment:
Generated new JSON Schema without any requirements on what specific values can be stored in certain variables (enums) that were based on a small example of results. This ensures that this schema can be used for multiple events/venues/etc.